src/components/Payment/CreatePaymentDialog.tsx

```diff
@@ -79,15 +79,15 @@ function CreatePaymentDialog(props: Props) {
   return (
     <Dialog open={props.open} fullScreen onClose={props.onClose} TransitionComponent={Transition}>
       <Box width="100%" maxWidth={900} padding="24px 36px" margin="0 auto">
-        <Typography variant="h5" component="h2" style={{ marginTop: 8, marginBottom: 8 }}>
-          Send funds {props.account.testnet ? <TestnetBadge style={{ marginLeft: 8 }} /> : null}
-        </Typography>
+        <MainTitle
+          title={<span>Send funds {props.account.testnet ? <TestnetBadge style={{ marginLeft: 8 }} /> : null}</span>}
```

Comment:
I added the badge component directly to the title prop so that it is always next to the title instead of floating on the right side if the device width is too small.